### PR TITLE
12-18-2020 equity dashboard chart data update

### PR DESCRIPTION
**Pull request by wpserviceuser**
Equity dashboard stats updates in this PR may be reviewed on staging: https://staging.covid19.ca.gov/equity/

After reviewing, if all looks well, approve and merge this Pull Request.

If there are issues with the data:

- Note concerns or issues here by commenting on this PR

- Work with Triston directly to resolve data issues

- Alert the COVID19 site team in Slack (in the Equity page channel)

### CODE REVIEW

File: data/reviewed/equitydash/equitytopboxdatav2.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+      "STATE_CASE_RATE_PER_100K": 4306.98773659852,
+      "CASE_RATE_PER_100K": 4658.98189325903,
       "POPULATION": 7290434,
-      "CASES_7DAYAVG_7DAYSAGO": 333353,
-      "RATE_DIFF_30_DAYS": 1030.19929952044
+      "CASES_7DAYAVG_7DAYSAGO": 339660,
+      "RATE_DIFF_30_DAYS": 1078.74236293752
     }
   ],
   "Demographics": [
```
Comment (coreycaitlin) [inline data/reviewed/equitydash/equitytopboxdatav2.json:12]: Not seeing fresh data in this file for any of the demographic groups.

**Review by coreycaitlin — COMMENTED** [review]
@TristonMosbacher @jbum Could you help track down what might have caused the demographic numbers in the `equitytopboxdatav2.json` file to *not* change? I think the related query is `casesAndDeathsByDemographic`, and the pre-Snowflake csv is `demog_rates_cumulative_YYYY-MM_DD.csv`.

File: data/reviewed/equitydash/equitytopboxdatav2.json (hunk 1 above)
Comment (jbum) [reply data/reviewed/equitydash/equitytopboxdatav2.json:12]: Not on snowflake either. Triston's gonna investigate, hopefully release the new data, and then I'll rerun the cron, which _should_ update the files on this ticket :)

Comment (vargoCDPH) [reply data/reviewed/equitydash/equitytopboxdatav2.json:12]: Are we waiting on something from Tristan on this?

Comment (coreycaitlin) [reply data/reviewed/equitydash/equitytopboxdatav2.json:12]: Waiting for the cron job to re-run. I think we should aim to merge tomorrow, not tonight.

Comment (vargoCDPH) [reply data/reviewed/equitydash/equitytopboxdatav2.json:12]: Agree. Sounds good. Thanks.

**Comment by abquirarte** [comment]
👍🏼

**Pull request closed without merging**